src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -1504,7 +1512,13 @@ public void saveStatistics(OutputStream out) throws IOException {
      * @throws ParseException
      */
     public void loadStatistics() throws IOException, ParseException {
-        loadStatistics(new File(getConfiguration().getStatisticsFilePath()));
+        String statisticsFilePath = getConfiguration().getStatisticsFilePath();
+        if(statisticsFilePath != null && !statisticsFilePath.isEmpty()){
+            File statisticsFile = new File(statisticsFilePath);
+            if(statisticsFile != null && statisticsFile.exists()){
+                loadStatistics(statisticsFile);
+            }
+        }
```

Comment:
I don't see the gain in this - if there is a problem with the file it raises an exception as expected - this is silently ignoring the problem and not logging anything. Change the test as I suggested instead of this check.